packages/uni_app/lib/model/providers/lazy/restaurant_provider.dart

```diff
@@ -13,9 +13,11 @@ class RestaurantProvider extends StateProviderNotifier<List<Restaurant>> {
   Future<List<Restaurant>> loadFromStorage(
     StateProviders stateProviders,
   ) async {
-    final restaurantDb = RestaurantDatabase();
-    final restaurants = await restaurantDb.getRestaurants();
-    return restaurants;
+    // TODO: remove this line after PR #1380 (fix: Added meals column to the RESTAURANTS table) is merged
+    return loadFromRemote(stateProviders);
+    // final restaurantDb = RestaurantDatabase();
+    // final restaurants = await restaurantDb.getRestaurants();
+    // return restaurants;
```

Comment:
Address this TODO before merging, please.

Reply:
This was discussed in a previous meeting. Since we are migrating to a different database, we decided not to modify the current implementation and to always fetch data from the remote in the meantime. However, if the new database isn’t ready for the next release, I will make the necessary changes to ensure everything functions properly.